cypress/objectsFile/loginActions.ts

```diff
@@ -0,0 +1,20 @@
+class Login{
+
+    openR365LoginPage(){
+        cy.visit('#/user/login');
+    }
+
+    fillUsernameField( username){
+        cy.get('[data-testid=userIdInputField]').type(username)
```

Comment:
you can use testing library commands https://testing-library.com/docs/cypress-testing-library/intro